docroot/themes/humsci/humsci_basic/src/js/main-nav/menu-toggle.js

```diff
@@ -0,0 +1,56 @@
+const menuToggle = document.querySelector('.su-main-nav__toggle');
+const mainMenu = document.querySelector('.su-main-nav__menu-lv1');
+const mobileNavBreakpoint = 992;
+let windowWidth = window.innerWidth;
+let isHidden;
+let wasDesktopSize;
+
+const collapseNav = () => {
+  menuToggle.setAttribute('aria-expanded', false);
+  mainMenu.setAttribute('aria-hidden', true);
+  menuToggle.innerHTML = "Menu";
+  isHidden = true;
+}
+
+const showNav = () => {
+  menuToggle.setAttribute('aria-expanded', true);
+  mainMenu.setAttribute('aria-hidden', false);
+  menuToggle.innerHTML = "Close";
+  isHidden = false;
+}
+
+if (menuToggle) {
+  // If below a certain breakpoint, hide the nav on page load
+  if (window.innerWidth < mobileNavBreakpoint) {
+    collapseNav();
+  }
+
+  // Toggle the nav when the the button is clicked
+  menuToggle.addEventListener('click', () => {
+    if (isHidden) {
+      showNav();
+    } else {
+      collapseNav();
+    }
+  });
+
+  // Handle the showing/hiding of the nav when resizing the browser
+  window.addEventListener('resize', () => {
+    windowWidth = window.innerWidth;
+
+    // When resizing from mobile to desktop, show the navigation
```

Comment:
can you add this comment here: 
// When resizing from mobile to desktop, ensure navigation is displayed, not hidden
  // If wasDesktopSize is false, it means we haven't gotten there yet and will to run this check   // Otherwise, if wasDesktopSize is true, we are above the mobileNavBreakpoint and don't need to keep showingNav